.github/workflows/build.yml

```diff
@@ -24,9 +24,9 @@ jobs:
            - os: windows
              ext: .exe
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
The github actions for checkout and upload/download artifacts had a version bump.  I just brought them up to current.